nano/templates/shipsensors.tmpl

```diff
@@ -1,6 +1,15 @@
+{{if data.viewing_silicon}}
+	<div class="block">
+		<div class="item">
+			<div class="itemContent bad">
+				Remote AI systems detected. Firewall protections forbid remote AI control.
+			</div>
+		</div>
+	</div>
+{{/if}}
```

Comment:
These blocks, and the data-part, feel somewhat copy-pasty. Hmm

Reply:
I couldn't figure out a way to handle this otherwise - NanoUI code looks like too much of a mess to try and find a way to have that inserted outside of the .tmpl files.